src/main/java/com/kim/dibt/core/utils/business/BusinessRule.java

```diff
@@ -0,0 +1,13 @@
+package com.kim.dibt.core.utils.business;
+
+import com.kim.dibt.core.utils.result.*;
+
+public class BusinessRule {
+    public static IResult run(IResult... logics) {
```

Comment:
IResultt türünden yerine Result Türünde olabilir ``
```suggestion
    public static IResult run(Result... logics) {
```